camel-kit-core/src/main/resources/skills/camel-migrate/guides/biztalk-phase1.md

```
@@ -0,0 +1,251 @@
+**Classify each adapter/component using `biztalk-component-mapping.md`:**
+- Mark adapters with known Camel equivalents
+- Mark adapters flagged as **ASK USER** (MSMQ, WCF-Custom, third-party, SAP, Dynamics, etc.)
+- Use dependencies from project files to pre-suggest replacement options (e.g., Azure SDK present → suggest `camel-azure-servicebus` for MSMQ)
+
+---
+
+### Step 1.2 — Resolve Proprietary Adapters
+
+For each adapter/component marked **ASK USER**, ask the user — one adapter at a time:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)